leaf-ui/index.html

```diff
@@ -17,6 +17,9 @@
         <link rel="stylesheet" type="text/css" href="./css/style.css" />
         <link rel="stylesheet" type="text/css" href="./css/analysis_style.css" />
 
+        <!--zebra css -->
```

Comment:
Add additional comments why zebra css is added.
For example: 
<!--zebra css - used for cycle detection component -->

Reply:
Zebra is used for creating Visually Appealing Dialog boxes?